### PR TITLE
Fixed Pulloquote Block border issue

### PR DESCRIPTION
**Pull request by viralsampat-multidots**
**Description**

Hello Team,

I have worked on issue #533 and resolved it from my end. Here, I have added my PR.

**Screenshots**

**Back-end**
backend

**Front-end**
frontend

**Testing Instructions**

- Go to page Type / to choose a block
- Select "Pullquote" block
- Add dummy text
- Click on the block setting and try to set the border.

**Contributors**

### CODE REVIEW

File: theme.json [hunk 1]
```diff
@@ -613,21 +613,21 @@
 			"core/pullquote": {
 				"border": {
 					"bottom": {
-						"style": "none",
-						"width": "0px"
+						"style": "solid",
```
Comment (luminuu) [inline theme.json]: In order to make the borders work again and keep the layout as in the design, the `bottom`, `left`, `right` and `top` attributes should be completely removed. This way there's no border when using the block as default, and borders can be added if the block settings are updated.

Please remove the mentioned attributes, but keep the `radius` attribute.

**Review by luminuu — CHANGES_REQUESTED** [review]

File: theme.json [hunk 2]
```diff
@@ -612,23 +612,9 @@
...
 					"radius": "var(--wp--preset--spacing--20)",
-					"right": {
-						"style": "none",
-						"width": "0px"
-					},
-					"top": {
-						"style": "none",
-						"width": "0px"
-					}
+					"style": "solid",
```
Comment (luminuu) [inline theme.json]: This does not need to be added, as the default design of the pullquote block does not have a border. Please remove these two lines.

**Review by luminuu — CHANGES_REQUESTED** [review]

File: theme.json (hunk 1 above)
Comment (viralsampat-multidots) [reply theme.json]: Hello @luminuu 

I have added my new PR.

Please review it and let me know your feedback for the same.

Thanks,

File: theme.json (hunk 2 above)
Comment (viralsampat-multidots) [reply theme.json]: Hi @luminuu 

Done,

Please check it

Thanks,

File: theme.json [hunk 3]
```diff
@@ -612,23 +612,8 @@
...
 					"radius": "var(--wp--preset--spacing--20)",
-					"right": {
-						"style": "none",
-						"width": "0px"
-					},
-					"top": {
-						"style": "none",
-						"width": "0px"
-					}
+					"style": "solid"
```
Comment (luminuu) [inline theme.json]: Please remove the `"style": "solid"` too, as it creates a border that is not in the design:

CleanShot 2023-10-04 at 13 07 38@2x

**Review by luminuu — CHANGES_REQUESTED** [review]

File: theme.json (hunk 3 above)
Comment (viralsampat-multidots) [reply theme.json]: Hi @luminuu 

I have updated my PR and removed the above-mentioned line.

Please review it.

Thanks,

**Review by luminuu — APPROVED** [review]
Tested. By default, the pullquote has no border as intended in the design, and borders can be added via the block settings if needed.

**Pull request merged**